playground/infrastructure/datastore_client.py

```diff
@@ -106,6 +110,35 @@ def save_to_cloud_datastore(self, examples_from_rep: List[Example], sdk: Sdk):
                 self._datastore_client.delete_multi(pc_objs_keys_for_removing)
                 logging.info("Finish of deleting extra playground examples ...")
 
+    def save_catalogs(self):
```

Comment:
This method is needed in order to load catalogs to the Cloud Datastore. It helps to decrease the coupling between python scripts and the router service